src/providers/ObsEditProvider.js

```diff
@@ -47,6 +49,25 @@ const ObsEditProvider = ( { children }: Props ): Node => {
     setUnsavedChanges( false );
   }, [] );
 
+  useEffect( () => {
+    const progressListener = EventRegister.addEventListener(
+      "INCREMENT_OBSERVATIONS_PROGRESS",
+      increments => {
+        setUploadProgress( currentProgress => {
+          increments.forEach( ( [uuid, increment] ) => {
+            currentProgress[uuid] ??= 0;
```

Comment:
@Chrischuck what is `currentProgress[uuid] ??= 0;` doing here? The `??=` operation seems to be causing e2e tests to fail, and I'm not sure why this line is necessary.